test/common/cdp.py

```diff
@@ -137,7 +137,10 @@ def cmd(self, cdp_port, env, show_browser, window_width, window_height,
         with open(os.path.join(profile, "handlers.json"), "w") as f:
             f.write('{"defaultHandlersVersion":{"en-US":4},"mimeTypes":{"application/xz":{"action":0,"extensions":["xz"]}}}')
 
-        cmd = [exe, "-P", "blank", f"--window-size={window_width},{window_height}", f"--remote-debugging-port={cdp_port}", "--no-remote", "localhost"]
+        env["MOZ_HEADLESS_WIDTH"] = str(window_width)
+        env["MOZ_HEADLESS_HEIGHT"] = str(window_height)
```

Comment:
This needs to be conditional, as I'm fairly sure that `--window-size` *does* apply to an interactive browser with `TEST_SHOW_BROWSER=1`. Can you please move it underneath the `if not show_browser:` in line 144?

Reply:
Ok, done!